book/src/configuration.md

```diff
@@ -11,8 +11,37 @@ This example file demonstrates the default configuration:
 [general]
 # Directories to search for templates, relative to the crate root.
 dirs = ["templates"]
+# Unless you add a `-` in a block, whitespace won't be trimmed.
+automatic_trim_block = false
 ```
 
+About `automatic_trim_block`, if you want to trim whitespace before or after a
+block, you need to use the minus sign. For example:
+
+```jinja
+<div>
+
+
+{%- if something %}
+Hello
+(% endif %}
+```
+
+In this case, only the content between `<div>` and `{%-` will be trimmed. If
+this option is enabled, all content before and after each jinja block will be
+trimmed.
```

Comment:
Then we introduce the option here, "In the template above, only the whitespace between `<div>` and `{%-` will be suppressed. If you enable `automatic_trim_block`, whitespace before and after each block will be suppressed by default. To preserve the whitespace, you can use the `+` operator:"